administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Card, H1, Spinner } from '@blueprintjs/core'
+import React, { useCallback, useContext } from 'react'
+import { useLocation, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import JsonFieldView from '../applications/JsonFieldView'
+import ErrorHandler from '../../ErrorHandler'
+import { useGetApplicationQuery, useVerifyApplicationVerificationMutation } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import NotFoundView from '../NotFoundView'
+import { useAppToaster } from '../AppToaster'
+
+const CARD_PADDING = 20
+
+const ApplicationViewCard = styled(Card)`
+  width: 800px;
+  margin: 10px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  padding: ${CARD_PADDING}px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  width: inherit;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+type ApplicationVerificationProps = {
+  applicationVerificationId: number
+  applicationAccessKey: string
+}
+
+const ApplicationVerification = ({ applicationVerificationId, applicationAccessKey }: ApplicationVerificationProps) => {
+  const config = useContext(ProjectConfigContext)
+  const appToaster = useAppToaster()
+  const showErrorToaster = () => appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+  const [verifyApplicationVerification, { loading: verificationLoading }] = useVerifyApplicationVerificationMutation({
+    onError: error => {
+      console.error(error)
+      showErrorToaster()
+    },
+  })
+
+  const submitApplicationVerification = useCallback(
```

Comment:
if you don't use the fkt in a lifecycle hook i think you can spare out the `useCallback`